src/promptflow/promptflow/_sdk/operations/_local_storage_operations.py

```diff
@@ -400,6 +387,29 @@ def persist_node_run(self, run_info: NodeRunInfo) -> None:
         filename = f"{str(line_number).zfill(self.LINE_NUMBER_WIDTH)}.jsonl"
         node_run_record.dump(node_folder / filename, run_name=self._run.name)
 
+    def load_node_run_info(self, parse_const_as_str: bool = False) -> List[NodeRunInfo]:
+        json_loads = json.loads if not parse_const_as_str else json_loads_parse_const_as_str
+        node_runs = []
+        for node_folder in sorted(self._node_infos_folder.iterdir()):
+            for node_run_record_file in sorted(node_folder.iterdir()):
+                if node_run_record_file.suffix.lower() != ".jsonl":
+                    continue
+                with read_open(node_run_record_file) as f:
+                    new_runs = [json_loads(line)["run_info"] for line in list(f)]
```

Comment:
Please add image path resolving logic to resolve the path in image dict to absolute path, so that it can be processed in the following logic.
![image](https://github.com/microsoft/promptflow/assets/10575286/65a6665a-f6a1-4cb3-8970-41ec3464fff0)

Reply:
Added resolve_multimedia_data_recursively to resolve image path to absolute path.